WalletWasabi.Gui/Controls/WalletExplorer/CoinListView.xaml

```diff
@@ -72,14 +75,17 @@
           <TextBlock Text="Merging unmixed coins with mixed ones undoes the mixes." Classes="warningMessage" IsVisible="{Binding LabelExposeCommonOwnershipWarning}" />
         </StackPanel>
       </StackPanel>
+      <Grid IsVisible="{Binding IsDetailsExpanded}" DockPanel.Dock="Bottom" HorizontalAlignment="Stretch" Margin="0,3">
+        <walexp:CoinDetailsGrid DataContext="{Binding SelectedCoin}" />
+      </Grid>
       <controls:BusyIndicator IsBusy="{Binding IsCoinListLoading}" Text="Loading...">
         <controls:ExtendedListBox Items="{Binding Coins}" VirtualizationMode="None" SelectedItem="{Binding Path=SelectedCoin, Mode=TwoWay}">
           <controls:ExtendedListBox.ItemTemplate>
             <DataTemplate>
-              <Grid>
+              <Grid Background="Transparent" Margin="0,0,0,4">
                 <Grid.ContextMenu>
                   <ContextMenu>
-                    <MenuItem Header="{Binding ExpandMenuCaption}" Command="{Binding ToggleDetails}">
+                    <MenuItem Header="{Binding Owner.ExpandedDetailsCaption}" Command="{Binding Owner.ToggleDetails}">
```

Comment:
Can we try to eliminate bindings with need for '.' in them?

I think put the command on the VM itself and let that call method on the parent?

Reply:
Actually the fact we had to do this, kind of hints that this Action (expand / hide) doesnt belong to a Coin, it belongs the to coinlist, so it doesnt make sense to have it in the coinlist exactly. Although it does work.

For example we are clicking on Coin A and saying Show Details (for Coin A) but if I click on Coin B, its now details for Coin B, I dont need to click on Show Details for Coin B.

My suggestion is that (and I know it was discussed earlier) we really would be better to have some kind of icon bottom right of the coin list.

However Im happy if you want to override me on this one :)

Reply:
![image](https://user-images.githubusercontent.com/4672627/72096381-dda3d000-32f8-11ea-8afe-8b8c1780082c.png)

![image](https://user-images.githubusercontent.com/4672627/72096654-61f65300-32f9-11ea-8d59-538ee989bc11.png)

You can quite easily make this change by adding something like:

```
  <Button Content="+" VerticalAlignment="Bottom" HorizontalAlignment="Right" ToolTip.Tip="Show / Hide Details" Command="{Binding ToggleDetails}" />
        </Grid>
```
underneath the controls.ExtendedListBox in CoinListView, and a few minor polishing steps after that.

I feel this gives a better user experience?

Reply:
These bindings, unfortunately, is the only clean way to go unless we go to another design of activating the details pane. Adding another indirection by adding a proxy method in CoinViewModel will only complicate things imho.

Reply:
> These bindings, unfortunately, is the only clean way to go unless we go to another design of activating the details pane. Adding another indirection by adding a proxy method in CoinViewModel will only complicate things imho.

I think then the design is wrong and *should* be changed before merging.